test/transformers/test_gemm_split_k_fp8_e4m3.py

```diff
@@ -0,0 +1,69 @@
+from test.utils import assert_verbose_allclose
+
+import pytest
+import torch
+
+from liger_kernel.ops.experimental.gemm_split_k_fp8_e4m3 import (
+    LigerFP8GemmSplitKFunction,
+)
+
+compute_capability = torch.cuda.get_device_capability(0)
+
+
+def to_float8_e4m3(x):
+    scale = x.abs().amax(1) / torch.finfo(torch.float8_e4m3fn).max
+    x = x.float() / scale.view(-1, 1)
+    return x.to(torch.float8_e4m3fn), scale
+
+
+@pytest.mark.parametrize(
+    "m, k, n",
+    [
+        (64, 64, 64),
+        (256, 256, 256),
+        (512, 512, 512),
+        (1024, 1024, 1024),
+        (64, 128, 64),
+        (256, 512, 256),
+        (512, 1024, 512),
+        (1024, 2048, 1024),
+    ],
+)
+@pytest.mark.parametrize(
+    "dtype, atol, rtol",
+    [
+        (torch.float32, 2e-1, 2e-1),
+    ],
+)
+@pytest.mark.skipif(
+    compute_capability not in [(8, 9), (9, 0)],
+    reason="FP8 GEMM is only supported on SM_89 and SM_90 CUDA devices.",
```

Comment:
we can change this to:
FP8 GEMM is only supported on SM_89 and newer architectures.

because we assume newer architectures will all support fp8